### PR TITLE
feat(tabset): add tabset component

### LINKED ISSUE #7: Tab
Recreate BS3 Tab with ng2 component

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**
Closes #7

### CODE REVIEW

File: src/tabset/tabset.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  @ContentChild(NgbTabTitle) titleTpl: NgbTabTitle;
+}
+
+@Component({
+  selector: 'ngb-tabset',
+  template: `
+    <ul class="nav nav-tabs" role="tablist">
+      <li class="nav-item" *ngFor="#tab of tabs; #i=index">
+        <a class="nav-link" [class.active]="i === activeIdx" (click)="select(i)">
+          {{tab.title}}<template [ngbTemplateOutlet]="tab.titleTpl?.templateRef"></template>
```
Comment (Foxandxss) [inline src/tabset/tabset.ts]: Wouldn't this allow both title input and titleTpl to exist both at once?

**Comment by Foxandxss** [comment]
My only complain is that you are using `1` and `0` to test the status of the active tab and we are using `true` and `false` already in other tests and I personally prefer the latter.

File: src/tabset/tabset.ts (hunk 1 above)
Comment (pkozlowski-opensource) [inline src/tabset/tabset.ts]: Yup, this is "by design", see: https://github.com/ng-bootstrap/core/pull/181/files#diff-f4f98e2db742c4772ab831ae9adb2dacR69

Having said this I don't mind either way so if you / anyone else prefers other logic, please let me know.

**Comment by pkozlowski-opensource** [comment]
Changing 0,1 to false,true.

**Comment by Foxandxss** [comment]
Works for me.

**Comment by pkozlowski-opensource** [comment]
Rebased on top of master / beta.16 and using built-in `ngTemplateOutlet` now. Please review.

**Pull request closed without merging**